cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -332,58 +314,9 @@ public void performAction(HttpRequest request, HttpResponder responder,
   }
 
   private void suspendResumeSchedule(HttpResponder responder, String namespaceId, String appId, String scheduleName,
```

Comment:
No need to have this method. Instead caller can directly call the `lifecycleService.suspendResumeSchedule`